### ISSUE TITLE
[Feature Request] Effect to toggle ON/OFF hotkeys

### ISSUE THREAD

**Issue by soycypher** [crowbartools/Firebot#2216]
**Describe the solution you'd like**
Be able to toggle on and Off the hotkeys
**Additional context**

**Comment by marrie** [comment]
Yes, or the ability to specify if the key should only be triggered when firebot is in focus or not in focus, or both, and a toggle on and off would be wonderful. I have not used hotkeys, however this would be a godsend
